src/app/pages/users/user/user.component.ts

```diff
@@ -234,11 +234,11 @@ export class UserComponent extends AbstractTabComponent implements OnInit {
           Validators.compose([
             Users.passwordWithNoSpace,
             Users.validatePassword,
-          ])),
+          ].concat(this.currentUserID && this.currentUserID !== '' ? [] : [Validators.required]))),
         repeatPassword: new FormControl('',
           Validators.compose([
             Users.validatePassword,
-          ])),
+          ].concat(this.currentUserID && this.currentUserID !== '' ? [] : [Validators.required]))),
```

Comment:
Better is to test for `stringFoo?.length === 0`. 
There's no `isEmptyString()` or similar helper in Utils.ts?

Reply:
> Better is to test for stringFoo?.length === 0.

stringFoo?.length === 0 doesn't give me the expected result
If the `currentUserID`is undefined, it will return me a `false` 

>  There's no isEmptyString() or similar helper in Utils.ts?

It seems not

Reply:
You're right. Let's keep the test as it is.